project/app/src/main/java/com/fillr/example/integration/activity/ExampleWebViewHeadlessActivity.java

```diff
@@ -48,6 +48,10 @@ protected void onCreate(Bundle savedInstanceState) {
 
         webView.setWebViewClient(new FillrWebViewClient());
 
+        //Alternatively if Affiliates are enabled,
+        webView.setWebViewClient(new AffiliateWebViewClient());

```

Comment:
Comment this line, so that the style is consistent with https://github.com/Fillr/android-example-integration/pull/18/files#diff-e9cbcb182d970ff76b3f2abad5978d53R148 ?

Reply:
fixed